pc_patch_ght.c:166:25: error: PCSCHEMA

Hi,
I have this error in build master branch: 
pointcloud/lib/pc_patch_ght.c:166:25: error: ‘PCSCHEMA {aka const struct <anonymous>}’ has no member named ‘x_position’
     if ( j == pa->schema->x_position || j == pa->schema->y_position )

This mi CMake Build configuration(GHT tags/v0.1.0):
-- The C compiler identification is GNU 5.4.0
-- The CXX compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Setting POINTCLOUD build type - RelWithDebInfo
-- Looking for include file stdint.h
-- Looking for include file stdint.h - found
-- Looking for include file getopt.h
-- Looking for include file getopt.h - found
-- Found LibXml2: /usr/lib/x86_64-linux-gnu/libxml2.so (found version "2.9.3") 
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.8") 
-- Found CUnit: /usr/lib/x86_64-linux-gnu/libcunit.so  
-- Found LibGHT: /usr/local/lib/libght.so  
-- Found LazPerf: /usr/local/include  
-- Configuring done
-- Generating done
-- Build files have been written to: /home/carlo/Downloads/UbuntuInstalls/pgpointcloud/pointcloud-build

--- a/lib/cunit/cu_pc_patch.c
+++ b/lib/cunit/cu_pc_patch.c
@@ -1200,7 +1200,7 @@ test_patch_set_schema_compression_ght()
 	pcfree(str);
 
 	// assign a schema with unknown dimension to the patch
-	pat1 = pc_patch_set_schema(pat0, simpleschema, 1, 0.0);
+	pat1 = pc_patch_set_schema(pat0, simpleschema, 0.0);
 	CU_ASSERT(pat1 != NULL);
 	str = pc_patch_to_string(pat1);
 	CU_ASSERT_STRING_EQUAL(str, "{\"pcid\":0,\"pts\":[[0.4,0.8,1.2,0],[0.3,0.6,0.9,0],[0.2,0.4,0.6,0],[0.1,0.2,0.3,0],[0,0,0,0]]}");

--- a/lib/pc_patch_ght.c
+++ b/lib/pc_patch_ght.c
@@ -132,8 +132,8 @@ pc_patch_ght_from_uncompressed(const PCPATCH_UNCOMPRESSED *pa)
 	pt.schema = pa->schema;
 	pt.readonly = PC_TRUE;
 
-	xdim = pa->schema->dims[pa->schema->x_position];
-	ydim = pa->schema->dims[pa->schema->y_position];
+	xdim = pa->schema->xdim;
+	ydim = pa->schema->ydim;
 
 	schema = ght_schema_from_pc_schema(pa->schema);
 	if ( ght_tree_new(schema, &tree) != GHT_OK ) {
@@ -162,11 +162,12 @@ pc_patch_ght_from_uncompressed(const PCPATCH_UNCOMPRESSED *pa)
 				GhtAttributePtr attr;
 				double val;
 
+				dim = pc_schema_get_dimension(pa->schema, j);
+
 				/* Don't add X or Y as attributes, they are already embodied in the hash */
-				if ( j == pa->schema->x_position || j == pa->schema->y_position )
+				if ( dim == pa->schema->xdim || dim == pa->schema->ydim )
 					continue;
 
-				dim = pc_schema_get_dimension(pa->schema, j);
 				pc_point_get_double(&pt, dim, &val);
 
 				ght_schema_get_dimension_by_index(schema, j, &ghtdim);

